modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -65,18 +65,27 @@ A click on the three vertical dots opens an actions menu. Actions can be perform
 
 === Spaces
 
-Spaces are organizational units to sort content and make it available to certain members.
+Spaces are special folders for teams. They are organized by the team members itself so that admins don't need to manage them. Even if members leave, the files stay in place so your team can keep on working.
 
-Under `Personal`, every user has a private space, while the `Spaces` area is intended for collaboration, i.e., sharing on a larger scale.
 
+Under `Personal`, every user can store private files, while the `Spaces` area is intended for collaboration, i.e., working together on a project, in a department or in a school class.
 
-In spaces, editors can do a little more and there's the additional role of manager.
+Members can be added to a Space so that they gain access to all files within the Space and see, whol else is a member of this Space. Within a Space, members can have different roles. By default, there are the roles:
```

Comment:
```suggestion
Members can be added to a Space so that they gain access to all files within the Space and see who else is a member of this Space. Within a Space, members can have different roles. By default, there are the roles:
```